### DISCUSSION TITLE
Is there any way to swc does transpile in cascating before exec like @babel/node?

### DISCUSSION THREAD

**Discussion by AugustoCalaca** [swc-project/swc#3835] [Q&A]
I would like to migrate all my codebase from babel to use only swc
I have one or two scripts which they use @babel/node to run them
These scripts have some imports to other local files and the script breaks if I transpile it with swc
```
// break cause does not transpile imported files by script.js before exec
swc script.js -o script.out.js && node script.out.js
```

```
// ok
babel-node --extensions \".js,.ts\" script.js
```

Is there any way to swc does transpile in cascating before exec like @babel/node?

**Comment by kwonoj** [comment]
swc cli doesn't offer to traverse modules. I think what you're looking for is either something like https://github.com/Brooooooklyn/swc-node, or https://typestrong.org/ts-node/docs/transpilers/#swc
